packages/jmix-front-generator/src/test/generators/sdk/sdk-generator.test.ts

```diff
@@ -55,27 +55,12 @@ describe('sdk generator test', () => {
     //TODO
   });
 
-  it('should fail writing if no model provided', (done) => {
```

Comment:
Why are we removing this test?

Reply:
In generators which based on defaultPipeline, we dont have access to `writing `method (it exists inside `defaultPipeline`). Plus, we will never have case, when writing method will run without model, this checking happens on stage `getProjectModel` (I added another test to check this case).